### ISSUE TITLE
Studio removes showFileName prop from xml when set to false in Component Inspector

### ISSUE THREAD

**Issue by fmcd** [cuba-platform/cuba#3147]
Cuba Studio 2021.2 build 7660.152

Create a screen with a file upload component.
Toggle showFileName property in Component Inspector.
When set to false, the showFileName attribute in removed from the <upload> xml tag.
However, file name is still shown by the component at runtime unless showFileName="false" is explicitly set on the component xml attributes.

**Comment by skatova** [comment]
Which version of the platform do you use?
Could you please provide more information about your problem and if it is possible, attached a small project that reproduced it?

**Comment by fmcd** [comment]
Thank you for reply.

I created a new project using Cuba Studio 2021.2 build 7660.152 with only a single screen with the file upload control, and the problem does NOT happen.

It seems the problem is only happening on an existing project created Cuba Studio 2019.2-13.2, which I had then opened using Cuba Studio 2021.2 build 7660.152.

I will try and strip down the live project that is showing the issue when I get a chance.
